server/lib/routes/post-csp.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Route to report CSP Violations to metrics
+ */
+
+var DEFAULT_RATE_LIMIT = 90;
+
+module.exports = function (options) {
+  options = options || {};
+
+  /**
+   * Rate limit 'rateLimitPercentage' % of messages.
+   * @param rateLimitPercentage
+   * @returns {boolean}
+   */
+  function isRateLimited(rateLimitPercentage) {
+    var RATE_LIMIT = rateLimitPercentage;
+    // rate limit can be 0
+    if (typeof rateLimitPercentage === 'undefined') {
+      RATE_LIMIT = DEFAULT_RATE_LIMIT;
+    }
+    var min = 0;
+    var max = 100;
+
+    var rand = min + Math.floor(Math.random() * (max - min + 1));
```

Comment:
Ahha, I see this code comes from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random to include both endpoints. Could this be made into its own function (probably preferable) or simplified since both min and max are hard coded:

```js
var rand = Math.floor(Math.random() * (100 + 1));
```